### PR TITLE
Move all SELinux policy pkgs to gitlab

### PR DESCRIPTION
**Pull request by ShawnHa0**
Move all selinux packages in order to avoid committing big binaries in github. Changed corresponding codes in selinux_setup.pm, using download link from gitlab now.

- Related ticket: https://progress.opensuse.org/issues/94982
- Needles: N/A
- Verification run: https://openqa.suse.de/tests/8367980#
- Related mr: 
    https://gitlab.suse.de/QA-APAC-I/testing/-/merge_requests/1
    https://gitlab.suse.de/QA-APAC-I/testing/-/merge_requests/2

### CODE REVIEW

**Review by Amrysliu — COMMENTED** [review]
I suggest to upload a tar package which includes all the selinux packages, then we only need one time 'wget'

**Review by lilyeyes — APPROVED** [review]
LGTM.

**Review by rfan1 — APPROVED** [review]
LGTM

File: tests/security/selinux/selinux_setup.pm [hunk 1]
```diff
@@ -59,16 +64,18 @@ sub run {
             'selinux-policy-minimum-20220124-150400.121.1.noarch.rpm',
             'selinux-policy-devel-20220124-150400.121.1.noarch.rpm'
         );
```
Comment (rfan1) [inline tests/security/selinux/selinux_setup.pm:66]: @ShawnHa0 
Given the fact that you have already put all pkgs in a tarball.
I suggest you remove these lines and later
``` #rpm -ivhU --nosignature --nodeps --noplugins ./*.rpm

File: tests/security/selinux/selinux_setup.pm [hunk 2]
```diff
@@ -59,16 +64,18 @@ sub run {
...
+        assert_script_run("wget --quiet --no-check-certificate $repo_link/$policy_pkg_20220124");
+        assert_script_run("tar -xzf $policy_pkg_20220124");
         foreach my $file (@files) {
-            assert_script_run 'wget --quiet ' . data_url("selinux/$file");
             assert_script_run("rpm -ivhU --nosignature --nodeps --noplugins $file");
         }
     }
     elsif (is_sle('>=15') && is_sle('<15-sp4')) {
         my @files
           = ('selinux-policy-20200219-3.6.noarch.rpm', 'selinux-policy-minimum-20200219-3.6.noarch.rpm', 'selinux-policy-devel-20200219-3.20.noarch.rpm');
```
Comment (rfan1) [inline tests/security/selinux/selinux_setup.pm:75]: same comment as above

**Review by lilyeyes — APPROVED** [review]
LGTM.

**Comment by jouyingbin** [comment]
@ShawnHa0 one comment, the subject, should be **SELinux**

**Comment by jouyingbin** [comment]
> @ShawnHa0 one comment, the subject, should be **SELinux**

git commit message should be adjusted too , thanks :)

**Review by rfan1 — APPROVED** [review]
LGTM

**Comment by ShawnHa0** [comment]
> > @ShawnHa0 one comment, the subject, should be **SELinux**
> 
> git commit message should be adjusted too , thanks :)

Will modify in my next rebase

**Review by Amrysliu — APPROVED** [review]

**Pull request merged**